### ISSUE TITLE
Waypoint mode is not framerate independent

### ISSUE THREAD

**Issue by Tokinsom** [skymen/follower#1]
[follower_waypoint_timescale_bug.zip](https://github.com/skymen/follower/files/12433902/follower_waypoint_timescale_bug.zip)
Hold Space to set timescale to 0.5, release to restore.

The distance between segments changes relative to the timescale when using waypoint mode.

**Comment by skymen** [comment]
Yeah, waypoint mode with negative speed goes to the "one waypoint per tick" method of path resolution.

Using positive speeds does a framerate independant path resolution. This method of "one waypoint per tick" is very useful for some effects, but yeah i suppose I could reimplement it in a way that keeps all its advantages but also becomes framerate independant
